docs/tools/qemu.md

```diff
@@ -109,151 +122,173 @@ configuration.
-/backstores/pscsi> create name=disk1 dev=/dev/sdf
-Note: block backstore recommended for SCSI block devices
-Created pscsi storage object disk1 using /dev/sdf
-/backstores/pscsi> cd /vhost
-/vhost> create
-Created target {==naa.5001405a160fe2e1==}.
-Created TPG 1.
-/vhost/naa.5001405a160fe2e1> cd /vhost/naa.5001405a160fe2e1/tpg1/luns
-/vhost/naa.50...2e1/tpg1/luns> create /backstores/pscsi/disk1
-Created LUN 0.
-/vhost/naa.50...2e1/tpg1/luns> cd /
-/> ls
-o- / ..................................................................... [...]
-  o- backstores .......................................................... [...]
-  | o- block .............................................. [Storage Objects: 0]
-  | o- fileio ............................................. [Storage Objects: 0]
-  | o- pscsi .............................................. [Storage Objects: 1]
-  | | o- disk1 ............................................ [/dev/sdf activated]
-  | |   o- alua ............................................... [ALUA Groups: 0]
-  | o- ramdisk ............................................ [Storage Objects: 0]
-  | o- user:fbo ........................................... [Storage Objects: 0]
-  | o- user:rbd ........................................... [Storage Objects: 0]
-  | o- user:zbc ........................................... [Storage Objects: 0]
-  o- iscsi ........................................................ [Targets: 0]
-  o- loopback ..................................................... [Targets: 0]
-  o- vhost ........................................................ [Targets: 1]
-    o- {==naa.5001405a160fe2e1==} .......................................... [TPGs: 1]
-      o- tpg1 .............................. [naa.500140565cd16730, no-gen-acls]
-        o- acls ...................................................... [ACLs: 0]
-        o- luns ...................................................... [LUNs: 1]
-          o- lun0 .............................. [pscsi/disk1 (/dev/sdf) (None)]
-/> exit
-```
-
-The World-Wide port name assigned by *targetcli* can then be used to specify the
-device to attach on QEMU command line.
-
-```plaintext
-# qemu-kvm (your options) \
-	-device pcie-root-port,bus=pcie.0,id=rp1 \
-	-device vhost-scsi-pci,wwpn={==naa.5001405a160fe2e1==},bus=pcie.0
-```
-
-The attached disk can then be seen from the guest OS using (for instance) the
-*lsscsi* command.
-
-```plaintext
-# lsscsi -g
-[0:0:0:0]    disk    ATA      QEMU HARDDISK    2.5+  /dev/sda   /dev/sg0
-[6:0:1:0]    zbc     HGST     HSH721415AL42M0  a250  /dev/sdb   /dev/sg1
-```
+To attach a zoned device to a virtual machine guest, you must first prepare a
+virtual TCM SAS adapter, using the *targetcli* tool. Specify the device to
+attach by using a block device file. The example below illustrates this
+operation for the disk `/dev/sdf`:
+
+1. **Use the targetcli Tool to Create a Device**
+   
+   Use *targetcli* to create the device that you will attach.
```

Comment:
Remove the leading spaces

Reply:
Hold on... Are the spaces needed due to the numbered list ?

Reply:
The spaces are needed for the numbered list.

Since there are two procedures here that are almost identical, it makes sense to present them in this parallel way and in a numbered list.